client/src/containers/CampaignPage.js

```diff
@@ -32,6 +33,17 @@ class CampaignPage extends Component {
     }
   }
 
+  calculateCampaignDuration = creationDateString => {
+    const creationDate = new Date(creationDateString);
+    const expireDate = new Date(creationDateString);
+    expireDate.setDate(expireDate.getDate() + MIN_CAMPAIGN_DURATION);
+    expireDate.setDate(expireDate.getDate() + (1 + 7 - expireDate.getDay()) % 7);
```

Comment:
given that math is hard, it would be a good idea to put some sort of explanation here, either in the form of a comment or a named variable for the expression

Reply:
good thought.  Ill add a blerb with what the goal of this date math is.